static/src/stylesheets/module/email/_form.scss

```diff
@@ -0,0 +1,220 @@
+// TODO These mixins are here while we run the A/B test as the styles are custom to the email form but if they become perm then
+// they should be made reusable
+
+@mixin rounded-input--inline-button() {
+    &text-input {
+        border-right: 0;
+        @include border-radius(1000px 0 0 1000px);
+        float: left;
+        width: 65%;
+    }
+
+    &submit-input {
+        font-size: 12px;
+        @include border-radius(0 1000px 1000px 0);
+        border-left: 0;
+        float: left;
+        margin-right: 0;
+        padding-left: 10px;
+        width: 35%;
+    }
+}
+
+.email-sub {
+    @include clearfix();
+}
+
+.email-sub__ { // Email subscription component namespace
+
+    /**
+     * Base Styles
+     */
+
+    &iframe {
```

Comment:
I guess from a code base consistency point of view, I should expand this out - anyone else have thoughts (@sndrs) ? I was slightly concerned about the readability too to be honest...